### PR TITLE
Chapter 10: Updating, showing, and deleting users

### PR DESCRIPTION
**Pull request by DucVinh2609**
61258162_416329695877109_2737554034972950528_n

### CODE REVIEW

**Comment by DucVinh2609** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -22,10 +28,53 @@ def create
...
+    @user = User.find_by id: params[:id]
+    redirect_to root_path unless current_user? @user
+  end
+
+  def admin_user
+    redirect_to root_path unless current_user.admin?
+  end
+
+  def load_user
+    @user = User.find_by id: params[:id]
```
Comment (phamvanbk11) [inline app/controllers/users_controller.rb]: Trùng code trên correct_user rồi nhé

**Comment by phamvanbk11** [comment]
commented

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (DucVinh2609) [reply app/controllers/users_controller.rb]: Em đã fix rồi ạ!

**Comment by DucVinh2609** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**